### PR TITLE
Onboarding MessageView, Setting item confirmation dialog, bug squashes

### PR DESCRIPTION
**Pull request by vincentbavitz**

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -919,6 +920,23 @@
       }
     };
 
+    // Set user's launch count.
+    const prevLaunchCount = window.getSettingValue('launch-count');
+    window.launchCount = !prevLaunchCount ? 1 : prevLaunchCount + 1;
```
Comment (msgmaxim) [inline js/background.js]: I wouldn't make it global until we actually use in more than one place.

File: preload.js [hunk 2]
```diff
@@ -62,6 +62,8 @@ window.CONSTANTS = {
   MAX_LOGIN_TRIES: 3,
   MAX_PASSWORD_LENGTH: 32,
   MAX_USERNAME_LENGTH: 20,
+  DEFAULT_PUBLIC_CHAT_URL: 'https://chat.lokinet.org',
```
Comment (msgmaxim) [inline preload.js]: Why can't we use `config.get('defaultPublicChatServer')` like before?

File: ts/components/session/LeftPaneMessageSection.tsx [hunk 3]
```diff
@@ -291,4 +396,61 @@ export class LeftPaneMessageSection extends React.Component<Props, any> {
       });
     }
   }
+
+  private handleJoinPublicChat() {
```
Comment (msgmaxim) [inline ts/components/session/LeftPaneMessageSection.tsx]: We shouldn't be making an almost exact copy of `handleJoinChannelButtonClick`, needs refactoring.

File: preload.js (hunk 2 above)
Comment (vincentbavitz) [reply preload.js]: Done.

File: ts/components/session/LeftPaneMessageSection.tsx (hunk 3 above)
Comment (vincentbavitz) [reply ts/components/session/LeftPaneMessageSection.tsx]: Done.

**Pull request merged**